algorithms/index.html

```diff
@@ -215,18 +374,16 @@
             </section>
             <section>
               <h2>Laundry v2 Code</h2>
-              <pre><code>
-            var piles = {'shirts': [], 'socks': []}
+                <pre><code>
+                    piles = {'shirts': [], 'socks': []}
```

Comment:
this is going to indent the slide improperly. the first line after `<pre><code>` should be all the way on the left-hand side or on the same line as the tags without any whtiespace. the following lines need to be indented less.

<img width="901" alt="screen shot 2016-07-23 at 9 42 14 am" src="https://cloud.githubusercontent.com/assets/2753545/17079050/c1aea642-50b9-11e6-8e24-ed6608fb546b.png">